foreman.te

```diff
@@ -495,6 +495,9 @@ optional_policy(`
 files_search_pids(httpd_t)
 stream_connect_pattern(httpd_t, foreman_var_run_t, foreman_var_run_t, foreman_rails_t)
 
+# Allow Apache access to static assets
+allow httpd_t foreman_lib_t:file { getattr map open read };
```

Comment:
Afaik there are symlinks under /var/lib/foreman/webpack/. Do they need special handling here?

Reply:
They didn't in my testing.

`/usr/share/foreman/public` (the docroot of Apache) is a symlink to `/var/lib/foreman/public`, and all things in `/var/lib/foreman/public` are `foreman_lib_t`

If the question was "is there any special handling needed to resolve the symlink itself", I don't see this being needed (the symlink is `usr_t`, which Apache seems to be able to access)

Reply:
Shouldn't we label the public directory's contents as `httpd_sys_content_t` instead?

Reply:
That's also possible (and has the same result in my setup).

Who would be responsible in doing the relabeling if we change it?

Reply:
Is this line still needed? I'd imagine that it's (mostly redundant).

Reply:
Nope, just me being lazy ;)